io/src/ply_io.cpp

```diff
@@ -391,9 +388,8 @@ pcl::PLYReader::vertexColorCallback (const std::string& color_name, pcl::io::ply
   {
     b_ = std::int32_t (color);
     std::int32_t rgb = r_ << 16 | g_ << 8 | b_;
-    memcpy (&cloud_->data[vertex_count_ * cloud_->point_step + rgb_offset_before_],
-            &rgb,
-            sizeof (pcl::io::ply::float32));
+    std::copy_n(&rgb, sizeof (pcl::io::ply::float32),
```

Comment:
Please recheck this entire file

Reply:
Reverted this file; these are all examples of `memcpy` that I left untouched elsewhere, where different values were being stuffed into a byte array